public/script.js

```diff
@@ -0,0 +1,100 @@
+'use strict';
+
+const range = 5;
+const instructionText = `Type a number between 1 and ${range}`;
+
+// STATE
+
+let currentScore;
+let guess = Number(document.querySelector('.guess-input').value);
+let highScore = 0;
+let secretNumber;
+
+// FUNCTIONS
+
+function isValidGuess() {
+  // When there is no input
+  if (!guess) {
+    setTextContent('message', 'Please type a number into the box on the left');
+    return false;
+  }
+  // When guess is not between 1 - (range)
+  if (guess < 1 || guess > range) {
+    setTextContent('message', `Number must be between 1 and ${range}`);
+    return false;
+  }
+  // When input is a decimal
+  if (guess !== Math.trunc(guess)) {
+    setTextContent('message', 'Guess must be a whole number (no decimals)');
+    return false;
+  }
+  return true;
+}
+
+function generateNewSecretNumber() {
+  secretNumber = Math.trunc(Math.random() * range) + 1;
+}
+
+function loseGame() {
+  currentScore = 0;
+  setTextContent('score', currentScore);
+  setTextContent('secret-number', secretNumber);
+  setTextContent('message', 'You lost the game 😱️');
+}
+
+function setTextContent(targetClass, newContent) {
+  return document.querySelector(`.${targetClass}`).textContent = newContent;
```

Comment:
```suggestion
  document.querySelector(`.${targetClass}`).textContent = newContent;
```
The returned value serves no purpose.